lib/AuditoManager.dart

```diff
@@ -52,53 +25,37 @@ class AudioManager {
         isPlaying = false;
```

Comment:
I'm not trying to be rude here. This bool doesn't work because the audio player will not loop audio by default, so `isPlaying` will be true even if the audio has stopped,